backend/src/main/java/de/ultical/backend/data/DataStore.java

```diff
@@ -813,4 +906,16 @@ private void modifyTeamAdmin(Team team, User admin, BiConsumer<Team, User> dbAct
             }
         }
     }
+    
+    public List<Roster> getRosterForPlayer(final DfvPlayer player) {

```

Comment:
@kaffeetrinkr - I don't get what this function is doing or if it was just a placeholder for something to come?!